src/components/Dialog/__stories__/Dialog.stories.mdx

```diff
@@ -349,17 +362,19 @@ We can set the triggers which will be responsible for hide the dialog
           <Dialog
             modifiers={modifiers}
             shouldShowOnMount
+            containerSelector={`#${HIDE_TRIGGERS_CONTAINER}`} // for testing purpose
```

Comment:
I think it problematic to add tests category to our main dialog doc (can be confusing). What about adding new Tests story which will contain tests stories for all relecant components (right now only dialog)

Reply:
I didn't see yet how jest works with those test stories but I think the best practice should be separating components instead of creating one big file in terms of rerunning specific tests on failure and reporting.

Reply:
Anyway, my second option would be to have a separate storybook just for testing, but I think that would take longer.
Regarding exposing a "Test" category - I don't see an issue about it

Reply:
let's talk about it in a team level then?